broker-util/oo-admin-repair

```diff
@@ -952,10 +952,13 @@ $chk_gear_mongo_node = true if fix_ssh_keys or fix_district_uids or fix_removed_
 $chk_district = true if fix_district_uids or fix_removed_nodes
 $chk_usage = true if fix_usage
 
-populate_user_hash
-populate_domain_hash
-populate_district_hash
-populate_app_hash
+# Skip populating hashes if --orphaned-envs is used
+if fix_orphaned_envs.nil?
```

Comment:
This logic is probably incorrect. Multiple flags can be passed into `oo-admin-repair`. So if I run `oo-admin-repair --consumed-gears --orphaned-envs`, the hashes will not be populated and the command will likely fail.